src/EprPrnIntegration.Api.UnitTests/UpdateProducersFunctionTests.cs

```diff
@@ -76,10 +75,20 @@ public async Task Run_NoUpdatedProducers_LogsWarning()
         _configurationMock.Setup(c => c["UpdateProducersStartHour"]).Returns(startHour.ToString());
```

Comment:
Can we remove this setup as we are not using this config anymore

Reply:
done